gladly-team/tab-web
# TFAC-294 Setting Target=_top for External Links

Modify the `Link` component:
* If the link URL is not on the current domain or the path does not start with `/newtab/`, use `target="_top"` by default
* If the link URL is on the current domain and the path starts with `/newtab/`, don’t set a default target

QA'd manually:
* the links conditionally add the target as expected
* when the app is in an iframe, external links navigate the top frame
* when the app is in an iframe, internal (same-app) links navigate in the nested frame
